.circleci/config.yml

```diff
@@ -124,3 +132,114 @@ jobs:
       - codecov/upload:
           file: languages/java/app-encryption/target/site/jacoco/jacoco.xml
           flags: languages_java_app_encryption
+  languages-csharp-logging:
+    docker:
+      - image: mcr.microsoft.com/dotnet/core/sdk:2.1
+    steps:
+      - checkout
+      - run:
+          name: Generate cache key
+          command: |
+            cd languages/csharp/
+            ./checksum.sh /tmp/checksum.txt ./Logging
+      - restore_cache:
+          keys:
+            # when pom changes, use increasingly general patterns to restore cache.
+            # vN prefix in case we ever need to regenerate all caches
+            - v1-nuget-{{ .Branch }}-{{ checksum "/tmp/checksum.txt" }}
+            - v1-nuget-{{ .Branch }}-
+            - v1-nuget-
+      - run:
+          name: Build
+          command: |
+            cd languages/csharp/Logging
+            ./scripts/clean.sh
+            ./scripts/build.sh
+      - run:
+          name: Tests
+          command: |
+            cd languages/csharp/Logging
+            ./scripts/test.sh
+      - save_cache:
+          paths:
+            - ~/.nuget
+          key: v1-nuget-{{ .Branch }}-{{ checksum "/tmp/checksum.txt" }}
+      - store_test_results:
```

Comment:
doesn't seem like this is working properly. according to https://circleci.com/docs/2.0/collect-test-data/ it seems like they need junit or cucumber format files. although not as critical as code coverage, it would still be nice if we can get the test result summary to show up in the circleci builds